WordPress/src/main/java/org/wordpress/android/designsystem/DesignSystemColorsScreen.kt

```diff
@@ -0,0 +1,118 @@
+package org.wordpress.android.designsystem
+
+import android.content.res.Configuration
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Divider
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.wordpress.android.R
+
+@Composable
+fun DesignSystemColorsScreen(
+    modifier: Modifier = Modifier
+) {
+    LazyColumn(
+        modifier = modifier,
+        horizontalAlignment = Alignment.Start,
+        verticalArrangement = Arrangement.spacedBy(
+            dimensionResource(id = R.dimen.reader_follow_sheet_button_margin_top)
+        )
+    ) {
+        item {
+            ColorTitle("Foreground")
+            val listForeground: List<ColorOption> = listOf(
+                ColorOption("Primary", MaterialTheme.colorScheme.primary),
+                ColorOption("Secondary", MaterialTheme.colorScheme.secondary),
+                ColorOption("Tertiary", MaterialTheme.colorScheme.tertiary),
+                ColorOption("Quartenary", MaterialTheme.colorScheme.quartenary),
+                ColorOption("Brand", MaterialTheme.colorScheme.brand),
+                ColorOption("Error", MaterialTheme.colorScheme.error),
+                ColorOption("Warning", MaterialTheme.colorScheme.warning),
+                ColorOption("WP", MaterialTheme.colorScheme.wp),
+                )
+            ColorCardList(listForeground)
+
+            ColorTitle("Background")
+            val listBackground: List<ColorOption> = listOf(
+                ColorOption("Primary", MaterialTheme.colorScheme.primaryContainer),
+                ColorOption("Secondary", MaterialTheme.colorScheme.secondaryContainer),
+                ColorOption("Tertiary", MaterialTheme.colorScheme.tertiaryContainer),
+                ColorOption("Brand", MaterialTheme.colorScheme.brandContainer),
+                ColorOption("WP", MaterialTheme.colorScheme.wpContainer),
+            )
```

Comment:
Add the Divider option here before merging

Reply:
@ravishanker Do you mean to make the last divider thicker? 
<img width="274" alt="Screenshot 2024-02-27 at 3 07 22 PM" src="https://github.com/wordpress-mobile/WordPress-Android/assets/7199140/381c1b9e-0671-437c-9a79-3a7dc1439927">

Reply:
Yes, I noticed that too but I don't see a definition for that in the [Figma](https://www.figma.com/file/LyzJJa6ANKToUswWmRrZee/%F0%9F%A7%B1-Jetpack-App-Design-System-iOS?type=design&node-id=102-186&mode=design&t=HR4dAG6naRMBnhmd-0) which is why I skipped that category. 

Compared to the Figma, the iOS implementation also has an extra color for `Success` and `Quartenary Foreground` (in addition to `Quartenary Background`), and is missing the `WP Foreground`, `WP Background` colors that are in the Figma.

I was using the Figma definitions as the source of truth, but let me know if I should be following the iOS implementation instead. Also wanted to mention that the colors on Android will likely need to be updated as Chris [mentioned](https://a8c.slack.com/archives/C04TXQU42HM/p1708599421092949) both platforms will have different color definitions.

Reply:
For now, let's just match the iOS.

Reply:
Hmm, I think it might make sense to check with Chris and Alp before making these changes. What do you think? 

I started taking a look at updating the values to match iOS and noticed that a few values seem incorrect (first screenshot) or are missing entirely (second screenshot). This is in addition to differing values from the Figma that I mentioned in the comment above, so there seems to be a few bugs and/or some confusion here. 

Screenshots from iOS:


<img width="353" alt="Screenshot 2024-02-27 at 5 50 29 PM" src="https://github.com/wordpress-mobile/WordPress-Android/assets/7199140/71a0d4f8-424d-4583-86c0-c8cb7fe98a22"> 

<img width="353" alt="Screenshot 2024-02-27 at 5 50 41 PM" src="https://github.com/wordpress-mobile/WordPress-Android/assets/7199140/4f8e228c-c932-4053-a23a-dce1ef207f65">